modules/ref_fb_module/src/renderer_fb_impl.cpp

```diff
@@ -823,33 +924,55 @@ void RendererFbImpl::getYMinMax(const SignalContext& signalContext, double& yMax
 
 void RendererFbImpl::renderAxis(sf::RenderTarget& renderTarget, SignalContext& signalContext, const sf::Font& font, bool drawXAxisLabels, bool drawTitle)
```

Comment:
Was the axis rendering checked with both array and scalar signals on the renderer at the same time? Do all the properties work (eg. SingleXAxis, SingleYAxis) in that case?